content/lessons/solid/solid-part-1.md

```diff
@@ -0,0 +1,31 @@
+---
+title: SOLID Principles Part 1
+template: lesson
+draft: false
+slug: /courses/solid-part-1
+course: SOLID
+tags:
+- Usability
+- Maintainability
+- OOP
+description: |
+  This lesson serves as an introduction to the SOLID Architecture Principles, going over when and how to implement them, in a multiple-choice quiz format. The tone is conversational: you will be guided towards why an answer is correct or incorrect as you select the different options.
+videoLinks:
+  - https://www.youtube.com/watch?v=yxf2spbpTSw
+  - https://www.youtube.com/watch?v=L2m-S0Pj_Xk
+  - https://www.youtube.com/watch?v=ObHQHszbIcE
+  - https://www.youtube.com/watch?v=-ptMtJAdj40
+  - https://www.youtube.com/watch?v=9oHY5TllWaU
+  - https://www.youtube.com/watch?v=xahwVmf8itI
+readingLinks:
+  - link: https://mari-azevedo.medium.com/s-o-l-i-d-principles-what-are-they-and-why-projects-should-use-them-50b85e4aa8b6
+    title: SOLID Principles- What are they and why projects should use them
+    description: 8 minute read
+  - link: https://en.wikipedia.org/wiki/SOLID
+    title: SOLID on Wikipedia
+  - link: https://fi.ort.edu.uy/innovaportal/file/2032/1/design_principles.pdf
+    title: “Design Principles and Design Patterns” by Robert C. Martin
+    description: This paper originated the SOLID principles.
+timeToCompletion: ~30 minutes
```

Comment:
I would bump this up to at least an hour/hour-and-a-half since the total length of just the videos is at least thirty minutes.